src/css/relay.css

```diff
@@ -37,52 +35,49 @@
 }
 
 .fx-relay-menu-iframe ::-moz-focus-inner,
-.fx-relay-menu-body ::-moz-focus-inner,
 .fx-relay-menu ::-moz-focus-inner,
 .fx-relay-icon ::-moz-focus-inner {
   border: 0;
 }
 
-.fx-relay-menu-body {
-  margin: 0;
-  padding: 0;
-  background: var(--relayGrey10);
-  background-color: var(--relayGrey10);
-  box-sizing: border-box;
-}
-
 .fx-relay-menu-iframe iframe { 
   border-radius: 4px;
   background: var(--relayGrey10);
   background-color: var(--relayGrey10);
   border: 0;
   box-shadow: 3px 3px 3px 3px #00000017;
   animation: fxRelayFadeIn 0.2s ease forwards;
+  position: relative;
+  z-index: 2;
```

Comment:
This `z-index` (and the `position` too) is no longer necessary. Switching the `::after` on the parent class to use a SVG instead of the triangle shadow hack solved this. 👍 

Removed in 05460091ed852a38c39cfd8650d0daebe466c80f